docs/technical/settings.rst

```diff
@@ -0,0 +1,19 @@
+Settings
+========
+
+This is the list of Django settings defined in the ``common.py`` modules of edx-platform.
+
+.. note::
+    Toggle settings, which enable or disable a specific feature, are more extensively documented in the :ref:`feature toggles <featuretoggles>` section.
```

Comment:
The original sentence could read as if the setting _also_ appears with more extensive details (as in appearing in both).

```suggestion
    Toggle settings, which enable or disable a specific feature, are documented in the :ref:`feature toggles <featuretoggles>` section.
```